activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api/src/main/java/org/activiti/cloud/modeling/api/ProjectConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.modeling.api;
+
+public interface ProjectConfiguration {
+
+    String getId();
+
+    void setId(String id);
+
+    void setEnableCandidateStarters(Boolean enableCandidateStarters);
+
+    Boolean getEnableCandidateStarters();
+
+    void setProject(Project project);
```

Comment:
Same here. Do we need to backward relation information?

Reply:
In this case, I use the Project here https://github.com/Activiti/activiti-cloud/pull/839/files#diff-77fa1dfa81b04014c0bc5c380c3b59c95e40a0783e56e57d3eb06672b77b205eR133 so I can't remove it